.therapist.yml

```diff
@@ -9,3 +9,5 @@ actions:
     include:
       - "*.css"
       - "*.less"
+  nsp:
+    run: yarn nsp check
```

Comment:
Would you rather have `yarn run lint:security` here too, as in the circleCI config? This way if it gets changed in the package.json, then it'll benefit both?

Reply:
I actually want to update circle in another PR to use therapist for all linting instead of yarn since it correctly identifies the files to lint unlike yarn.